frontend/src/components/browse/BrowseContent.js

```diff
@@ -179,77 +203,101 @@ export default function BrowseContent({
 
       <Divider className={classes.mainContentDivider} />
 
-      <TabContent value={tabValue} index={0}>
-        {filtersExpanded && tabValue === 0 && (
-          <FilterContent
-            className={classes.tabContent}
-            type={TYPES_BY_TAB_VALUE[0]}
-            applyFilters={handleApplyNewFilters}
-            filtersExpanded={filtersExpanded}
-            unexpandFilters={unexpandFilters}
-            possibleFilters={possibleFilters(TYPES_BY_TAB_VALUE[0], filterChoices)}
-          />
-        )}
-        {state && state.items && state.items.projects && state.items.projects.length ? (
-          <ProjectPreviews
-            projects={state.items.projects}
-            loadFunc={loadMoreProjects}
-            hasMore={state.hasMore.projects}
-            parentHandlesGridItems
-            className={classes.itemsContainer}
-          />
-        ) : (
-          <NoItemsFound type="organizations" />
-        )}
-      </TabContent>
-      <TabContent value={tabValue} index={1} className={classes.tabContent}>
-        {filtersExpanded && tabValue === 1 && (
-          <FilterContent
-            className={classes.tabContent}
-            type={TYPES_BY_TAB_VALUE[1]}
-            applyFilters={handleApplyNewFilters}
-            filtersExpanded={filtersExpanded}
-            unexpandFilters={unexpandFilters}
-            possibleFilters={possibleFilters(TYPES_BY_TAB_VALUE[1], filterChoices)}
-          />
-        )}
-        {state && state.items && state.items.organizations && state.items.organizations.length ? (
-          <OrganizationPreviews
-            organizations={state.items.organizations}
-            loadFunc={loadMoreOrganizations}
-            hasMore={state.hasMore.organizations}
-            showOrganizationType
-            parentHandlesGridItems
-          />
-        ) : (
-          <NoItemsFound type="organizations" />
-        )}
-      </TabContent>
-      {!hideMembers && (
-        <TabContent value={tabValue} index={2} className={classes.tabContent}>
-          {filtersExpanded && tabValue === 2 && (
+      <>
+        <TabContent value={tabValue} index={0}>
+          {filtersExpanded && tabValue === 0 && (
+            <FilterContent
+              className={classes.tabContent}
+              type={TYPES_BY_TAB_VALUE[0]}
+              applyFilters={handleApplyNewFilters}
+              filtersExpanded={filtersExpanded}
+              unexpandFilters={unexpandFilters}
+              possibleFilters={possibleFilters(TYPES_BY_TAB_VALUE[0], filterChoices)}
+            />
+          )}
+
+          {/*
+            Render the spinner until results are returned.
+            Render the not found page if the object came back empty.
+          */}
+          {isLoading ? (
```

Comment:
We might need 2 different types of `isLoading` here: one for filters and one for just loading more projects. When applying a filter I would expect the spinner to **replace** the projects and when loading new projects I would expect the spinner to **appear under** the projects.